commit 83838ad36e826a70b44dfb957d27cd72f0090edc (refs/changes/80/1080/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-07T11:55:17-08:00 (2 years, 10 months ago)
    
    DOCKER-983 'docker --tls pull nope.example.com/nope' is way too slow

diff --git a/package.json b/package.json
index 9fe57a7..255dcac 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "bunyan": "1.8.1",
     "cmdln": "3.2.1",
     "dashdash": "1.10.0",
-    "docker-registry-client": "3.2.4",
+    "docker-registry-client": "3.2.5",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "expiring-lru-cache": "2.1.0",
     "extsprintf": "1.2.0",
